runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -47,8 +47,6 @@ decl_storage! {
 		///
 		/// The `Option<()>` is effectively a bool, but it never hits storage in the `None` variant
 		/// due to the guarantees of FRAME's storage APIs.
-		///
-		/// If this is `None` at the end of the block, we panic and render the block invalid.
 		Included: Option<()>;
```

Comment:
This makes no sense anymore.

Reply:
We still use it to ensure it's not included twice.